pygmt/tests/test_which.py

```diff
@@ -14,19 +14,19 @@ def test_which():
     """
     for fname in ["tut_quakes.ngdc", "tut_bathy.nc"]:
         cached_file = which(f"@{fname}", download="c")
```

Comment:
```suggestion
        cached_file = which(fname=f"@{fname}", download="c")
```

Ensure that the `fname` keyword parameter still works.